app/services/wakes/pageview_count_updater_service.rb

```diff
@@ -8,14 +8,14 @@ def initialize(location)
 
   def update_pageview_count
     if pageviews_since_last_update.positive?
-      update_location_pageview_count && update_resource_aggregate_count && update_wakeable_aggregate_count
+      (update_location_pageview_count && update_resource_aggregate_count && update_wakeable_aggregate_count).tap do
```

Comment:
@benhutton This ensures the result of all three method calls gets returned, which is the intended behavior of this method.